### DISCUSSION TITLE
Kodi 19.3 and Kodi Plex Connect (stable repo) works ?

### DISCUSSION THREAD

**Discussion by BartOon** [croneter/PlexKodiConnect#1716] [Q&A] [answered]
Hi folks,

is anybody works with the latest kodi 19.3 version and plex kodi connect (stable repo) ?
I got empty librairies when I updated
first I was thinking it was an data corruption I recovered from a backup but none
I installed from scratch, plex kodi connect was not able to connect to PMS
I restaured my (Windows) computer to a previous point, it go back to normal so I updated kodi to 19.3, issue come back, I downgraded to 19.1, I have some data corruption but librairies are good and sync is ok

**Comment by croneter** [comment]
First time I hear of this. Try resetting the Kodi database (in the PKC settings under Advanced) after update.

**Comment by BartOon** [comment]
Ok I re(do), just in case

To be consistent, you have to go in wheel > add-ons > my add-ons > All > PlexKodiConnect > Configure > Advanced > Reset data ... (in red)

I did but the same comportment, reset is effective, it asked me all first setup questions, I saw the PMS and folders selection, more weirdo is the windows appears different after and before sync,

![image](https://user-images.githubusercontent.com/3241239/142736650-e5a9b
![Screenshot 2021-11-20 130529](https://user-images.githubusercontent.com/3241239/142736703-0efae4af-e3b8-4fb1-a5bf-f349a1164e3d.png)
![Screenshot 2021-11-20 130414](https://user-images.githubusercontent.com/3241239/142736705-8e6d67f2-2418-493d-8a44-1436d5aa4dc7.PNG)
![Screenshot 2021-11-20 130618](https://user-images.githubusercontent.com/3241239/142736707-bd570ea5-419e-45b2-b307-b461522f3b20.png)
d2f-a117-418d-b08d-05865f8206b4.png)
And I am able to see file in add-on > PKC > open > etc.

I goes to add-on again, to check update the 3.1.0 was the latest, I try to figure it out to have 3.5, but won't. I check dependencies to see, one was not latest, and failed to update, I disable PKC, to update goes well, and PKC shown 3.5 version I installed, but same issue at the end, libraries are empty.

but in case I will figure it out, if you tell me everything is fine, I will reset all the installation and go back from scratch

I don't know if something with my config, it's 2 users setup, and it work well until now

**Comment by BartOon** [comment]
Outch, I goes from scratch as soon I put password on profile (wheel > Profiles > Profiles > Master user > Lock Preferences > Master lock > Numeric password, I set one and I let all options lock * by default, all come empty
Something happens with the new Kodi release ?

**Reply by BartOon** [reply]
If I remove the pass it goes back to normal

**Comment by croneter** [comment]
I can't really follow your arguments, sorry. To sum up, I understand the following:
* You experienced issue while updating Kodi from 19.x to 19.3. Libraries suddenly appeared empty. 
* You were unable to install the latest PKC version (which is extremely important!) because they did not show up. 
Were you able to solve these issues? Don't even start on the masterlock before you've solved these...

Install the latest Kodi 19.3 from scratch, do NOT update. Then install the PKC repo and latest PKC. Does that work?

Next time, open a Github issue please.

**Comment by BartOon** [comment]
that’s exactly that, I have issue with the latest Kodi 19.3 and the latest PKC 3.5.8, if I update or installed from scratch.
I have been testing so this is why I noticed PKC was not automatically updated when I updated Kodi, perhaps a delay, but I finally did.

But whatever it looks the issue come from the masterlock, I can open an Github issue, I will want confirm I not the only one to have the problem first, did you able to reproduce ?

**Comment by croneter** [comment, marked as answer]
Please open a new issue and share your DEBUG log files, see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by BartOon** [comment]
for records; https://github.com/croneter/PlexKodiConnect/issues/1718
